src/FEValues/common_values.jl

```diff
@@ -30,28 +26,22 @@ end
 end
 
 """
-    reinit!(cv::CellValues, x::Vector)
-    reinit!(bv::FaceValues, x::Vector, face::Int)
+    reinit!(cv::CellValues, x::Vector, cell::Union{AbstractCell,Nothing}=nothing)
+    reinit!(fv::FaceValues, x::Vector, face::Int, cell::Union{AbstractCell,Nothing}=nothing)
```

Comment:
Stupid questions, but why don't we write
```suggestion
    reinit!(cv::CellValues, x::Vector)
    reinit!(fv::FaceValues, x::Vector, face::Int)
    reinit!(cv::CellValues, x::Vector, cell::AbstractCell)
    reinit!(fv::FaceValues, x::Vector, face::Int, cell::AbstractCell)
```
for the user docs? I think having cell=nothing by default is an implementation detail.